app/scripts/views/sign_up.js

```diff
@@ -230,54 +230,43 @@ function (Cocktail, _, p, BaseView, FormView, Template, Session, AuthErrors,
 
     _initAccount: function () {
       var self = this;
-      var email = self.$('.email').val();
-      var password = self.$('.password').val();
-      var customizeSync = self.$('.customize-sync').is(':checked');
+
       var preVerifyToken = self.relier.get('preVerifyToken');
+      var account = self.user.initAccount({
+        email: self.$('.email').val(),
```

Comment:
At least the email should be updated to use [getElementValue](https://github.com/mozilla/fxa-content-server/blob/cdd999e395d560f67762c3b3d258859e5852902a/app/scripts/views/form.js#L301-L312) so that leading/trailing whitespace is trimmed.